runtime/lua/vim/ui.lua

```diff
@@ -71,6 +71,11 @@ end
 ---     - highlight (function)
 ---               Function that will be used for highlighting
 ---               user inputs.
+---     - secret (boolean|nil)
+---               If `true`, the user's input will be shown as
+---               a series of asterisks ("*") (defaults to `false`).
```

Comment:
The asterisks are the default impl, right?

```suggestion
---               (default: false) If `true`, user input is hidden (displayed as asterisks "*" by default) instead of echoed.
```

Reply:
This is from the `inputsecret` docs so it seems like the asterisks can't be changed:
```
a) the user's response will be displayed as a sequence of asterisks ("*") thereby keeping the entry secret, and
```